chapters/statements.tex

```diff
@@ -392,7 +392,7 @@ \subsection{When-Statements}\label{when-statements}
 Merging the \lstinline!when!-statements can lead to less efficient code and different models with different behavior depending on the order of the assignment to \lstinline!y1! and \lstinline!y3! in the algorithm.
 \end{example}
 
-\subsubsection{Restrictions on Where a When-statement may occur}\label{restrictions-on-where-a-when-statement-may-occur}
+\subsubsection{Where a When-statement may occur}\label{restrictions-on-where-a-when-statement-may-occur}\label{where-a-when-statement-may-occur}
```

Comment:
Hmm, here "when" should be a keyword in "when-statement" but it is capitalized while "may occur" is not.

Reply:
Will capitalize 'may occur'. I think capitalizing When-statement makes sense; as it's a compound and not the keyword on its own.

Reply:
Resolving.